src/com/datacash/training/chapter13/answers.md

```diff
@@ -0,0 +1,52 @@
+1) D. All of the above
+
+2) No, only objects can be used as type arguments.
+
+3)
+```java
+public class FlightSched<T, V> {
+```
+
+4)
+```java
+public class FlightSched<T, V extends Thread> {
+```
+
+5)
+```java
+public class FlightSched<T, V extends T> {
+```
+
+6) `?` is the wildcard argument, which can be used when the exact type is not known/
+
+7) Yes, the wildcard argument can be bounded.
+
+8) 
+```java
+public T MyGen ( T type ) {

```

Comment:
This is embarrassing because I implemented this correctly in some changes I recently made to HPS. The correct form is:
```java
public <T> T MyGen ( T type ) {
```